uint8 slave_number
string CS_cmd
uint8 interpolation_time_period
int8 interpolation_time_index
int32[] value
---
bool result